pkg/kubelet/cm/deviceplugin/device_plugin_handler.go

```diff
@@ -61,12 +61,14 @@ type Handler interface {
 type HandlerImpl struct {
 	// TODO: consider to change this to RWMutex.
 	sync.Mutex
+	// devicePluginManager is an implementation of deviceplugin.Manager interface.
 	devicePluginManager Manager
 	// activePods is a method for listing active pods on the node
 	// so the amount of pluginResources requested by existing pods
 	// could be counted when updating allocated devices
 	activePods ActivePodsFunc
-	// devicePluginManagerMonitorCallback is used for testing only.
```

Comment:
if this is indeed used for testing only we should probably retain this..

Reply:
@BenTheElder thanks, I think this `Callback` is not for test now, @jiayingz WDYT?